mw-server/db/query/ways.query.sql

```diff
@@ -240,3 +240,47 @@ RETURNING *,
 -- name: DeleteWay :exec
 DELETE FROM ways
 WHERE uuid = $1;
+
+-- name: GetWayDetailsById :many
+SELECT
+    w.uuid AS way_uuid,
```

Comment:
let's be convenient and use full table names

Reply:
done